service-deployment/clusterconfig-example.yaml

```diff
@@ -117,7 +115,9 @@ clusterinfo:
     prometheus_port: 9090
     # port for node exporter
     node_exporter_port: 9100
-  
+    # cadvisor port
```

Comment:
Why put this configuration in grafana?

Reply:
Put it in promethues. This config currently is used by prometheus. The web shows the config is folded.

Reply:
Sorry, I didn't expand all the code. 😆